pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -309,3 +357,25 @@ func TestMakePod(t *testing.T) {
 		})
 	}
 }
+
+func TestMakeWorkingDirScript(t *testing.T) {
+	for _, c := range []struct {
+		desc        string
+		workingDirs []string
+		want        string
+	}{{
+		desc:        "simple",
+		workingDirs: []string{"/workspace/foo", "/workspace/bar", "/baz"},
+		want:        "mkdir -p /workspace/foo /workspace/bar",
+	}, {
+		desc:        "empty",
```

Comment:
nit: maybe add a test that checks for workingDirs and returns empty

Reply:
Sorry, I'm not following (maybe because I'm tired and it's pretty late here). Can you restate/give an example?

Reply:
something like this ? https://gist.github.com/chmouel/b1b085d96ffd4b7a620c26ed3708462c

Reply:
Done, thanks for clarifying 😄